### DISCUSSION TITLE
Lesson 6 FAILED tests/test_fund_me.py::test_can_fund_and_withdraw

### DISCUSSION THREAD

**Discussion by DarrenWayn** [smartcontractkit/full-blockchain-solidity-course-py#711] [Q&A] [answered]
<img width="718" alt="Screen Shot 2022-01-06 at 17 38 08" src="https://user-images.githubusercontent.com/22322944/148370042-6d88ea5d-17df-4145-b824-933f406bb6ac.png">
FAILED tests/test_fund_me.py::test_can_fund_and_withdraw 

is there anything wrong? the test failed

**Comment by cromewar** [comment, marked as answer]
Hello @DarrenWayn you must use accounts.add with 'S' and you have it as account.add, that is giving you an error. Give it a try ;)

**Reply by DarrenWayn** [reply]
Yes i missed it thanks @cromewar

**Reply by cromewar** [reply]
I'm glad is solved now.
